### PR TITLE
[gatsby-source-wordpress] add polylang support

### PR DESCRIPTION
**Pull request by maru3l**
Add Polylang-rest-api support to Gatsby-source-wordpress plugin.

I have also make the polylang_translations link between entities.

### CODE REVIEW

File: packages/gatsby-source-wordpress/src/__tests__/normalize.js [hunk 1]
```diff
@@ -57,6 +57,10 @@ describe(`Process WordPress data`, () => {
       { id: 2, acf: {} },
     ])
   })
+  it(`Creates links between entities and polylang translations entities`, () => {
+    entities = normalize.mapPolylangTranlations(entities)
```
Comment (pieh) [inline packages/gatsby-source-wordpress/src/__tests__/normalize.js]: I don't think it test anythign given that entities here don't have `polylang_translations` field

Comment (maru3l) [reply packages/gatsby-source-wordpress/src/__tests__/normalize.js]: @pieh Sorry, it's the first time I'm doing testing.

I've add `Polylang_translations` in `data.json` and change the snapshot. I've tried to inspired myself with the other test who link two entities together.

File: packages/gatsby-source-wordpress/src/normalize.js [hunk 2]
```diff
@@ -277,6 +277,24 @@ exports.mapElementsToParent = entities =>
     return e
   })
 
+exports.mapPolylangTranlations = entities =>
+  entities.map(entitie => {
```
Comment (pieh) [inline packages/gatsby-source-wordpress/src/normalize.js]: just small nitpicky comment here - please change singular `entitie` to `entity`

**Comment by maru3l** [comment]
@pieh I've just change the variable name as asked and the test passed on my side. I've looked at the issue Travis error and it's something in the installation process.

**Comment by pieh** [comment]
Thanks @maru3l!

**Comment by DSchau** [comment]
@maru3l @pieh just a heads up, there's a minor typo in this PR that is probably worth resolving.

https://github.com/gatsbyjs/gatsby/pull/7724/files#diff-48de6c73f0ef63fe29ba87436e66bc41R280

`mapPolylangTranlations` -> `mapPolylangTranslations`

@maru3l any interest in a future PR to fix that up? Thank you!

**Pull request merged**